### PR TITLE
DelayedDebit rules

### PR DESCRIPTION
**Pull request by ptbrowne**
Implement multi rules for delayed debit.

### CODE REVIEW

File: src/ducks/settings/DelayedDebitAlertRules.jsx [hunk 1]
```diff
@@ -162,6 +154,14 @@ const DumbDelayedDebitSettingSection = props => {
   )
 }
 
+DelayedDebitCard.propTyps = {
```
Comment (drazik) [inline src/ducks/settings/DelayedDebitAlertRules.jsx]: `propTyps` => `propTypes`

File: src/ducks/settings/DelayedDebitAlertRules.jsx [hunk 2]
```diff
@@ -73,28 +73,20 @@ const getModalProps = ({ initialDoc, t }) => ({
...
-      : null
-  }
+const getCheckingsDefaultValue = props => {
+  const selectedCreditCard = getCreditCardDefaultValue(props)
+  return selectedCreditCard && selectedCreditCard.checkingsAccount
+    ? selectedCreditCard.checkingsAccount.data
+    : null
+}
 
+class DelayedDebitCard extends React.Component {
```
Comment (drazik) [inline src/ducks/settings/DelayedDebitAlertRules.jsx]: Nit: converting it to a functional component (so we tend to use hooks in it in the future) must be kinda free now

File: src/ducks/settings/DelayedDebitAlertRules.jsx (hunk 1 above)
Comment (ptbrowne) [reply src/ducks/settings/DelayedDebitAlertRules.jsx]: Not used anymore.

File: src/ducks/settings/DelayedDebitAlertRules.jsx (hunk 2 above)
Comment (ptbrowne) [reply src/ducks/settings/DelayedDebitAlertRules.jsx]: Not used anymore ;)

**Pull request merged**